packages/Discretization/src/DTK_PointInCell_def.hpp

```diff
@@ -12,16 +12,41 @@
 
 #include <DTK_DBC.hpp>
 #include <DTK_PointInCellFunctor.hpp>
+#include <DTK_Topology.hpp>
 #include <Kokkos_Core.hpp>
 
 namespace DataTransferKit
 {
+// Because search is static, we cannot use a private function so put the
+// function in its own namespace.
```

Comment:
I am not following you here.  Would you explain some more?

Reply:
The function `search` is static so it cannot call a `private` function because you would need to create the object.